modules/mobile/app/controllers/mobile/v0/facilities_info_controller.rb

```diff
@@ -6,10 +6,21 @@ class FacilitiesInfoController < ApplicationController
       SORT_METHODS = %w[home current alphabetical appointments].freeze
 
       def index
```

Comment:
I had been planning to modify the index method to work for both functions but ended up not doing so because there are a few unknowns that gave me pause. Specifically, the existing code looks for va treatment facilities and cerner facilities. The new endpoint replaces code that only uses the va treatment facilities. Also, here we need to exclude children and search for `schedulable: nil` because we need the info regardless of whether the facility is schedulable. Lastly, the other endpoint has a required sort param in the path, which isn't necessary here and only complicates matters. 

As a result, I decided to separate this into its own method and decided that it should be the index, so I renamed the other method. I chose to use the same schema, despite not needing all the data, because I didn't want another facility data representation and because it may make it easier for us to combine these later.